prov/efa/src/rxr/rxr_init.c

```diff
@@ -429,6 +429,28 @@ int rxr_get_lower_rdm_info(uint32_t version, const char *node,
 	return ret;
 }
 
+static void rxr_shm_space_check(void)
```

Comment:
The function name is ```check```, but it does disable shm. I would prefer, to change it 

```
    if (!rxr_shm_space_check())
         rxr_env.enable_shm_transfer = 0;
```

Reply:
also the function name can be ```rxr_shm_enough_space()``` to be more specific

Reply:
> The function name is `check`, but it does disable shm. I would prefer, to change it
> 
> ```
>     if (!rxr_shm_space_check())
>          rxr_env.enable_shm_transfer = 0;
> ```

Updated.
But I prefer leaving the function name as it is. And use 0 to indicate successful return, and error number to indicate failure.